tests/bsim/compile.source

```diff
@@ -42,6 +42,10 @@ function _compile(){
   local map_file_name=${exe_name}.Tsymbols
 
   local this_dir=${WORK_DIR}/${app}/${exe_basename}
+  local app_absolute=$(realpath "${ZEPHYR_BASE}/${app}")
+  if [ "${app_absolute}" != "${ZEPHYR_BASE}"* ]; then
```

Comment:
```suggestion
  if [[ "${app_absolute}" != "${app_root}"* ]]; then
```
Did you forget the double bracket? (otherwise this is always true for me)

Reply:
Yeah my bad 😅